### PR TITLE
added metadata for vivenio

### PR DESCRIPTION
**Pull request by iCodr8**

### CODE REVIEW

File: meta/vivenio/cms-plugin-contao/de.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+de:
+    title: vivenio Teilnehmerregistrierung
+    description: >
+        Fügt die vivenio Teilnehmerregistrierung zu ihrer Contao Webseite hinzu.
```
Comment (xchs) [inline meta/vivenio/cms-plugin-contao/de.yml]: Hier sollte die Du-Form verwendet werden.

**Comment by xchs** [comment]
Maybe you could make the logo available in SVG format?

**Comment by aschempp** [comment]
> Maybe you could make the logo available in SVG format?

Please be aware that **only** SVG logos are supported!

**Comment by iCodr8** [comment]
Thank you @Toflar

**Pull request merged**